ansible.cfg

```diff
@@ -16,6 +16,13 @@ host_key_checking = False
 retry_files_enabled = False
 retry_files_save_path = ~/ansible-installer-retries
 nocows = True
+remote_user = root
```

Comment:
We tend to control this via ansible_ssh_user being set in the inventory, do you not set that?